src/games/PlayerVsComputerGame.js

```diff
@@ -20,13 +20,13 @@ PlayerVsComputerGame.prototype.play = function(e) {
       this.ui.displayTie();
   } else {
       this.playerMarker = this.GameBoard.switchMarker(this.playerMarker);
-      PlayerVsComputerGame.prototype.simpleAI(this.ui, this.GameBoard, this.playerMarker);
+      PlayerVsComputerGame.prototype.pickRandomSpaceForAI(this.ui, this.GameBoard, this.playerMarker);
```

Comment:
I think this should actually end up being it's own `Computer` Class that doesn't know about the UI  and simply just returns the index of the spot on the board it wants to go. Your game can handle actually placing the spot on the board!